docs/process/bazel.md

```diff
@@ -58,6 +59,21 @@ causes jasmine to exit with a non-zero exit code.
 
 While testing, you can remove the `shard_count` attribute to prevent sharding and the `flaky`
 attribute to prevent repetition.
+Setting `--test_output=streamed` will disable sharding and `--flaky_test_attempts=1` will disable
+the reruns of tests that have been marked as `flaky`.
+
+A complete command for running a test with support for remote debugging is:
+
+```
+yarn bazel test \
+  //packages/angular/cli:angular-cli_test \
+  # Force local execution and disable sharding (also, immediately show test output):
+  --test_output=streamed \
+  # Disable reruns of failed flaky tests:
+  --flaky_test_attempts=1 \
+  # Pass `--inspect-brk` to the nodejs process running the test suite:
+  --test_arg=--node_options=--inspect-brk
```

Comment:
There's a `debug` configuration that enables most of those options (but not all)

https://github.com/angular/angular-cli/blob/d87822c1cef972026e7ea52be7728e7076478911/.bazelrc#L9

```
yarn bazel test //packages/angular/cli:angular-cli_test --config=debug
```

I'm not sure about the relevance of `flaky_test_attempts`.

Reply:
Nice! That makes this command a lot simpler.

`flaky_test_attempts` might be worth adding to that config but at worst it means having to quickly press `Ctrl+C` if/when the debugged test process exits to unblock the next build/re-run.

Reply:
Replaced the manual command with a recommendation to use the `--config=debug` flag.